### DISCUSSION TITLE
Sub-regions

### DISCUSSION THREAD

**Discussion by nataliefaith2** [code4recovery/12-step-meeting-list#1256] [General]
Is there a way of showing regions and sub-regions? 

View the website to see regions and sub-regions within the "Anywhere" tab.
[https://al-anon.org.au/meetings/](https://al-anon.org.au/meetings/ )

We want area members to know what their local region/area is and the office they can contact if they need support and I think this would be great to be visible on the meeting feature so new members get familiar with their region/area.

**Comment by SIA-WEBSUP** [comment]
I'm unsure what the request is.

Currently, you can select Region or a sub-region.
For instance, you can select NNSW to see everything in that entire region: https://al-anon.org.au/meetings/?region=nnsw
OR you can select a sub-region within that region to filter on that particular sub-region: https://al-anon.org.au/meetings/?region=nnsw-armadale

Could you be more specific on what else you would like to see?

BTW, I think the Z-order is off on your footer, since the dropdown is drawn 'under' the footer and difficult to use. It's easy enough to select 'Anywhere' to get access to the entire dropdown again. You might see if you can set the z-order of the footer to be higher.
